### ISSUE TITLE
Duplicated members of the same inherited base

### ISSUE THREAD

**Issue by marehr** [doxygen/doxygen#7027]
The following code:

```c++
//!\brief A Class
struct A
{
    /*!\name Group 1 */
    //!\{
    //!\brief A member
    int member;
    //!\}
};

//!\brief B Class
struct B : A
{};

//!\brief C Class
struct C : A
{};

//!\brief D Class
struct D : A
{};

//!\brief E Class
struct E : B, C, D
{};
```

Produces:
![image](https://user-images.githubusercontent.com/103778/58882122-de79e100-86db-11e9-8268-67d1e2d289b5.png)

Doxygen file is: [Doxyfile.txt](https://github.com/doxygen/doxygen/files/3252558/Doxyfile.txt)

I used Doxygen 1.8.15.

I would have expect that inheriting multiple time the same base class would result in declaring `member` only once in the derived class.

**Comment by albert-github** [comment]
Description looks like to be complete though it would always be nice to have also an attachment with a, small, self contained example (source+configuration file in a tar or zip) that allows us to reproduce the problem, saves the necessity to cut and paste.

**Comment by marehr** [comment]
Sorry, I didn't know. It would be helpful to create an [ISSUE_TEMPLATE](https://help.github.com/en/articles/manually-creating-a-single-issue-template-for-your-repository) that walks you through your preferences in reporting issues.

**Comment by marehr** [comment]
Okay interesting! Were seems to be a difference between using DOT and not DOT:

### Without DOT:
I think here you see that doxygen does not recognize that it uses the same base class multiple times.

![image](https://user-images.githubusercontent.com/103778/58888266-6b766780-86e7-11e9-9dee-eddc785da867.png)

Download: [issue7027_1.zip](https://github.com/doxygen/doxygen/files/3252953/issue7027_1.zip)

### With DOT:
But, it seems that DOT recognizes it?

![image](https://user-images.githubusercontent.com/103778/58888243-65808680-86e7-11e9-9118-eb7bc826104f.png)

Download: [issue7027_2.zip](https://github.com/doxygen/doxygen/files/3252955/issue7027_2.zip)

**Comment by albert-github** [comment]
I think the difference is just because of the limited possibilities of the drawing software used for the non-dot version. The dot package is far more sophisticated.

**Comment by albert-github** [comment]
Seen the fact that there is a good alternative in the form of the `dot` package and the fact that it would take very much time and knowledge about generating graphs I think best is to set the issue to won't fix.
